crypto/crypto.go

```diff
@@ -611,11 +644,7 @@ func AddressFromProgram(program []byte) types.Address {
 // 2. If no ma provides, it returns Sig delegated LogicSig
 // 3. If both sk and ma specified the function returns Multisig delegated LogicSig
 func MakeLogicSig(program []byte, args [][]byte, sk ed25519.PrivateKey, ma MultisigAccount) (lsig types.LogicSig, err error) {
```

Comment:
There is an unfortunate problem. This function is deprecated and has been replaced by `MakeLogicSigAccountEscrow`, `MakeLogicSigAccountDelegated`, and `MakeLogicSigAccountDelegatedMsig`.

The good news is that `MakeLogicSigAccountDelegated` and `MakeLogicSigAccountDelegatedMsig` call into `MakeLogicSig`, so this change works for them. The bad news is that `MakeLogicSigAccountEscrow` does not call into `MakeLogicSig`, and even if we modified it to do so, it does not currently return an error response.

Reply:
ugh, my IDE does not show that it has been deprecated, I should probably mark it deprecated as golang syntax.

I think I should be a nice dude to add the check in `MakeLogicSigAccountEscrow `, so the basic check should be reflected...

Reply:
never mind, I just realized `MakeLogicSigAccountEscrow` returns only `LogicSigAccount`, *without error*. That is unfortunate.

Reply:
Right, at the moment I think the only solution is to deprecate `MakeLogicSigAccountEscrow` and create a new function that will return an error too.

Reply:
deprecation change reflected in [f8dab81](https://github.com/algorand/go-algorand-sdk/pull/366/commits/f8dab818abe3564d6a577a18f6ea926357cebf62), let me know if you are good with new method name

Reply:
Thanks, new name looks good